runelite-client/src/main/java/net/runelite/client/ui/overlay/components/InfoBoxComponent.java

```diff
@@ -28,49 +28,75 @@
 import java.awt.Dimension;
 import java.awt.FontMetrics;
 import java.awt.Graphics2D;
+import java.awt.Image;
 import java.awt.Point;
 import java.awt.Rectangle;
-import java.awt.image.BufferedImage;
-import java.util.Objects;
+import lombok.Getter;
 import lombok.Setter;
-import net.runelite.client.ui.overlay.RenderableEntity;
 
 @Setter
-public class InfoBoxComponent implements RenderableEntity
+public class InfoBoxComponent implements LayoutableRenderableEntity
 {
-	private static final int BOX_SIZE = 35;
 	private static final int SEPARATOR = 2;
```

Comment:
Maybe this should be reduced to 1 or even none at all?

Reply:
That is padding from border. and the border is 2px wide, so, it is placed like right at the corner, i was thinking about making it 3 even.